package.json

```diff
@@ -50,8 +52,10 @@
     "react-map-gl": "^5.2.7",
     "react-player": "^2.6.1",
     "read-more-react": "^1.0.10",
+    "request-ip": "^2.1.3",
     "styled-jsx": "^3.3.0",
     "sugar": "^2.0.6",
+    "swr": "^0.3.5",
```

Comment:
If we decide to use a new module like https://www.npmjs.com/package/swr for remote data fetching, it might be generally used everywhere and not just for one single API call.